design/KruizeConfigurationProcedure.md

```diff
@@ -49,13 +49,15 @@ data:
         "hbm2ddlauto": "update",
         "showsql": "true"
       },
-      "cloudwatch": {
-        "accessKeyId": "",
-        "secretAccessKey": "",
-        "region": "",
-        "logGroup": "kruize-logs",
-        "logStream": "kruize-stream",
-        "logLevel": "INFO"
+      "logging": {
```

Comment:
@kusumachalasani - Manifest yamls don't need an update to include logging?

Reply:
yes, they need them. let me create an another PR!